Makefile

```diff
@@ -754,6 +754,14 @@ test-go-unit-tbot:
 		| tee $(TEST_LOG_DIR)/unit.json \
 		| gotestsum --raw-command -- cat
 
+# Runs tsh unit tests
+.PHONY: test-go-unit-tsh
+test-go-unit-tsh: FLAGS ?= -race -shuffle on
+test-go-unit-tsh:
+	$(CGOFLAG) go test -cover -json $(FLAGS) $(ADDFLAGS) ./tool/tsh/... \
+		| tee $(TEST_LOG_DIR)/unit.json \
+		| gotestsum --raw-command -- cat
+
```

Comment:
This target is more or less the same as `test-go-tsh`. Were you able to use that instead?

Reply:
Ah, missed that one probably. I've removed it.